scripts/prepare_sector_network.py

```diff
@@ -2789,6 +2791,261 @@ def add_electricity_distribution_grid(n, costs):
 #         constant=co2_limit,
 #     )
 
+def add_industry_demand(n, demand):
+
+    avail = spatial.nodes.intersection(demand.index)
+
+    # Low temperature demand between 50C and 150C
+    carrier = "low temperature heat 50-150C for industry"
+
+    n.madd(
+        "Bus",
+        avail + " " + carrier,
+        carrier=carrier
+    )
+
+    n.madd(
+        "Load",
+        avail + " " + carrier,
+        bus=avail + " " + carrier,
+        carrier=carrier,
+        p_set=demand.loc[avail, "demand(50-150C)[MW]"]
+    )
+
+    # Medium temperature demand between 150C and 250C
+    carrier = "medium temperature heat 150-250C for industry"
+
+    n.madd(
+        "Bus",
+        avail + " " + carrier,
+        carrier=carrier
+    )
+
+    n.madd(
+        "Load",
+        avail + " " + carrier,
+        bus=avail + " " + carrier,
+        carrier=carrier,
+        p_set=demand.loc[avail, "demand(150-250C)[MW]"]
+    )
+
+
+def add_egs_industry_supply(n, supply_curve):
+
+    # annuitize capex
+    supply_curve['capex[$/MW]'] = (
+        supply_curve['capex[$/MW]'] * 0.07 / (1 - (1 + 0.07) ** - 25)  # 7% interest rate, 25 years lifetime

```

Comment:
good to move these calculations in the prepare-egs file.
0.07 and 25 cannot stay here in the code, they must be parameters loaded from config